storops/unity/resource/lun.py

```diff
@@ -267,6 +300,10 @@ def delete(self, async_mode=False, force_snap_delete=False,
 
         if self.is_thin_clone:
             TCHelper.notify(self, ThinCloneActionEnum.TC_DELETE)
+
+        end = time.time()
+        log.info(
+            'Time used of lun {} deletion: {}'.format(self.name, end - start))
         return resp
```

Comment:
`resp` would change if using async mode. It doesn't impact the `delete` because `delete` doesn't return anything valuable. But for an operation like `create` or `attach`, the created object or hlu number will return. Then how to get the created object or hlu number? You'll need to change all the caller's codes.

Reply:
Need to figure out what is the response and how to handle it when implementing async create, attach or other opertions.